core/src/main/scripts/importer/validateData.py

```diff
@@ -249,7 +249,7 @@ class Validator(object):
     REQUIRE_COLUMN_ORDER = True
     ALLOW_BLANKS = False
 
-    def __init__(self, study_dir, meta_dict, portal_instance, logger):
+    def __init__(self, study_dir, meta_dict, portal_instance, logger, relaxed_mode):
```

Comment:
Please adjust `DataFileTestCase.validate()` in `core/src/test/scripts/unit_tests_validate_data.py` to this change in the constructor’s signature, as currently it makes every single unit test fail.